pkgs/by-name/op/open-webui/package.nix

```diff
@@ -5,6 +5,7 @@
   python312,
   nixosTests,
   fetchurl,
+  ffmpeg,
```

Comment:
Perhaps we could use `ffmpeg-headless` ?

Reply:
Just pushed an update that uses `ffmpeg-headless` :+1: